### PR TITLE
Update linters to conform to ShakaCode standards

### PR DESCRIPTION
**Pull request by robwise**
This change is in parallel with the change made for the webpack
tutorial.

See https://github.com/shakacode/react-webpack-rails-tutorial/pull/159

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -6,12 +6,46 @@ extends: eslint-config-airbnb
 plugins:
   - react
 
+globals:
+  __DEBUG_SERVER_ERRORS__: true
+  __SERVER_ERRORS__: true
+  google: true
```
Comment (justin808) [inline .eslintrc]: remove lines 12-14

**Comment by robwise** [comment]
@dylangrafmyre I'm pretty sure we're failing the linters in CI because of needing to update our eslint dependencies. I pass with the latest version of eslint (v1.10.1) locally, but the docker container is [shrinkwrapped to v1.6.0](https://github.com/shakacode/docker-lint/blob/master/npm-shrinkwrap.json#L454). How do I go about updating this?

**Comment by dylangrafmyre** [comment]
@robwise just need to update the package.json and npm shrinkwrap  followed by a push to origin(shakacode/docker-lint). This will trigger a new build of the container image on Docker Hub. And when CI runs it will pull down the latest build. I can help with this if needed?

**Comment by robwise** [comment]
Thanks @dylangrafmyre for helping me on this! We had to update the linter dependencies in the docker image. This should be good to go pending a final review.

**Comment by justin808** [comment]
Looks good. Let's merge.

**Comment by justin808** [comment]
@robwise Did you confirm that the generated code passes these linter changes?

**Comment by robwise** [comment]
@justin808 affirmative, good to go

**Pull request merged**